admin_manual/configuration/server/occ_command.rst

```diff
@@ -1587,17 +1623,16 @@ Ransomware Protection
 Use these commands to help users recover from a Ransomware attack.
 You can find more information about the application :doc:`in the documentation <security/ransomware-protection>`.
 
-.. note:: Ransomware Protection (which is an Enterprise app) needs to be installed and enabled to be able to use these commands.
+.. note::
+  Ransomware Protection (which is an Enterprise app) needs to be installed and enabled to be able to use these commands.
```

Comment:
How about linking "Ransomware Protection" to the app?

Reply:
Good idea, do we have a doc link available in the RWP document?
In case, please let me know what I have to copy in and I will do.

Reply:
doc:\`../../enterprise/ransomware-protection/index\` should do it.

Reply:
added